server/routes/items.js

```diff
@@ -298,12 +292,22 @@ router.get(
 router.post(
   '/sandbox/item/reset_login',
   asyncWrapper(async (req, res) => {
-    const { itemId } = req.body;
-    const { plaid_access_token: accessToken } = await retrieveItemById(itemId);
-    const resetResponse = await plaid.sandboxItemResetLogin({
-      access_token: accessToken,
-    });
-    res.json(resetResponse.data);
+    try {
+      const { itemId } = req.body;
+      const { plaid_access_token: accessToken } = await retrieveItemById(
+        itemId
+      );
+      const resetResponse = await plaid.sandboxItemResetLogin({
+        access_token: accessToken,
+      });
+      res.json(resetResponse.data);
+    } catch (error) {
+      if (error.response.status === 400) {
```

Comment:
You probably want to check for `error.response && error.response.status === 400` here just to be safe